### PR TITLE
Test that reliably fails with Artemis 2.1.0

### PR DESCRIPTION
**Pull request by andr3ej**

### CODE REVIEW

**Comment by adagys** [comment]
Looks like a useful test, but if, as you say, it takes 2 mins on average, it's probably not worth it.

**Comment by andr3ej** [comment]
next build (3rd one) should succeed

**Comment by andr3ej** [comment]
temporarily back to 2.1.0 so i can observe how may trials are really necessary in TC (4th build onwards)

**Comment by andr3ej** [comment]
@adagys Apparently TC needs just as many trials as my dev machine, so I've changed the test to version-check Artemis so it behaves like current master (1 trial) until the next time Artemis is upgraded. And then whoever is upgrading Artemis will have to deal with it.

File: client/rpc/src/integration-test/kotlin/net/corda/client/rpc/RPCStabilityTests.kt [hunk 1]
```diff
@@ -218,22 +218,47 @@ class RPCStabilityTests {
 
     @Test
     fun `client reconnects to rebooted server`() {
+        val trials = if (ArtemisConstants::class.java.`package`.implementationVersion == "1.5.3") 1 else 25
```
Comment (adagys) [inline client/rpc/src/integration-test/kotlin/net/corda/client/rpc/RPCStabilityTests.kt]: can you add a comment here explaining that there's a race condition with Artemis 2 and why we need multiple trials? And maybe a TODO to remove the trials once we resolve the flakiness

Comment (andr3ej) [inline client/rpc/src/integration-test/kotlin/net/corda/client/rpc/RPCStabilityTests.kt]: Done

**Pull request merged**